lib/dl_formula/dl_formula/definitions/functions_markup.py

```diff
@@ -275,6 +275,30 @@ class FuncBr(FuncMarkup):
     variants = make_variants("br")
 
 
+class FuncTooltip2(FuncMarkup):
+    name = "tooltip"
+    arg_cnt = 2
+    arg_names = ["text", "tooltip"]
+    argument_types = [
+        # tooltip(markup|str, markup|str)
+        ArgTypeSequence([MARKUP_EFFECTIVELY, MARKUP_EFFECTIVELY]),
+    ]
+    variants = make_variants("tooltip")
+    scopes = Function.scopes & ~Scope.SUGGESTED & ~Scope.DOCUMENTED
+
+
+class FuncTooltip3(FuncMarkup):
+    name = "tooltip"
+    arg_cnt = 3
+    arg_names = ["text", "tooltip", "placement"]
+    argument_types = [
+        # tooltip(markup|str, markup|str, str)
+        ArgTypeSequence([MARKUP_EFFECTIVELY, MARKUP_EFFECTIVELY, DataType.STRING]),
+    ]
+    variants = make_variants("tooltip")
+    scopes = Function.scopes & ~Scope.SUGGESTED & ~Scope.DOCUMENTED
+
+
```

Comment:
Why not create a base class as we always do?

Reply:
> Why not create a base class as we always do?

fixed